### ISSUE TITLE
Standardization of log messages from the Audit App

### ISSUE THREAD

**Issue by marcohald** [nextcloud/server#26127]
**Is your feature request related to a problem? Please describe.**
It would be easier to parse the Audit Log if all Entries have the same format.
Now i need to use a lot of regex to parse some entries in the audit.log file.
A standardized entry Format like Action: "What happened" would be nice

**Describe the solution you'd like**
A Format like in the Files Module would be great
https://github.com/nextcloud/server/blob/7ba89c13e70c030ac7b1100084eb10bebc1d15d4/apps/admin_audit/lib/Actions/Files.php#L58

The entries from this Line https://github.com/nextcloud/server/blob/7ba89c13e70c030ac7b1100084eb10bebc1d15d4/apps/admin_audit/lib/Actions/Sharing.php#L49 are hard to parse
Edit:
for example
`				'%s shared via link: \""%s" with ID "%s" has been shared via link with permissions "%s" (Share ID: %s)\"',`

**Comment by szaimen** [comment]
Hi there, do you mind creating a PR that improves this for discussion?
Thank you!

**Comment by brlin-tw** [comment]
I would also like this to be implemented, if not already.
